.github/workflows/pipelinePR.yml

```diff
@@ -0,0 +1,80 @@
+# Pipeline for Pull Requests
+#
+# Lint, Unit Test, Build, Deploy and Cleanup
+#
+name: Pipeline - Pull Request
+
+on:
+  pull_request:
+    types:
+      [opened, reopened, synchronize, closed]
+
+jobs:
+  lint-and-test:
+    name: Lint and Unit Test
+    if: github.event.pull_request.state == 'open'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run lint
+      - run: npm run test:ci
+
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Variables
+        shell: bash
+        run: |
+          echo -e "\n\n--- Environment Variables ---\n"
+          echo "HOME: ${HOME}"
+          echo "GITHUB_WORKFLOW: ${GITHUB_WORKFLOW}"
+          echo "GITHUB_RUN_ID: ${GITHUB_RUN_ID}"
+          echo "GITHUB_RUN_NUMBER: ${GITHUB_RUN_NUMBER}"
+          echo "GITHUB_ACTION: ${GITHUB_ACTION}"
+          echo "GITHUB_ACTIONS: ${GITHUB_ACTIONS}"
```

Comment:
question: what's the diff between GITHUB_ACTION and GITHUB_ACTIONS?

Reply:
The first is an ID of some kind while the other changes from true/false depending on whether the Action is currently running or not.  Untuitive!  ;)